modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/converter/ObjectFieldDTOConverter.java

```diff
@@ -63,7 +67,21 @@ public ObjectField toDTO(
 				indexedLanguageId = objectField.getIndexedLanguageId();
 				label = LocalizedMapUtil.getLanguageIdMap(
 					objectField.getLabelMap());
-				listTypeDefinitionId = objectField.getListTypeDefinitionId();
+
+				setListTypeDefinitionExternalReferenceCode(
```

Comment:
@murilostodolni  Based on this [commit](https://github.com/liferay-forms/liferay-portal/commit/db8e5e429a8e3af0a177eea2d5655eb6aa4c4921), we should move the `setListTypeDefinitionExternalReferenceCode` to the end, after `type = ObjectField.Type.create(objectField.getDBType());`.